Feat/ranking

SEOKKAMONI opened the pull request:
## 📄 Summary
> 
Ranking List 완성


<br>

## 🔨 Tasks
- 


<br>

## 🙋🏻 More

SEOKKAMONI committed:
SET :: Ranking 세팅

SEOKKAMONI committed:
ADD :: Ranking List Component

gimhanul reviewed (commented):
- [x] RankingList component props 사용해서 변경
- [x] Ranking component 삭제
    -> 페이지 만들 때 필요하고 현재는 필요 없습니당
- [x] 현재 element의 className을 다 대문자로 해놨는데, [css 네이밍 규칙](https://uipac.com/451)을 참고하여 클래스명 수정

이 필요한 것 같습니당~

SEOKKAMONI committed:
FIX :: 리뷰 반영
- RankingList component props 사용해서 변경
- Ranking component 삭제
- CSS 네이밍 규칙 참고하여 수정

gimhanul commented on src/App.js:
```diff
@@ -1,6 +1,15 @@
 function App() {
   return (
     <div className="App">
+      <RankingList 
+        ranking={4}
+        user_name={"김석진"}
+        git_id={"SEOKKAMONI"}
+        commit={144}
+      />
```
해당 컴포넌트는 Ranking Page 구현시 사용하면 되니 여기서는 삭제하는 게 좋을 것 같습니다~

gimhanul commented on src/components/Ranking/RankingList.js:
```diff
@@ -0,0 +1,29 @@
+import "./rankingList.scss";
+
+function RankingList( {ranking, user_name, git_id, commit} ) {
```
변수는 카멜케이스로 씁니다.
[변수 네이밍](https://programmers-sosin.tistory.com/entry/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EB%B3%80%EC%88%98-%EC%84%A0%EC%96%B8-%EB%B0%8F-%EA%B7%9C%EC%B9%99-JavaScript-Variable)에 관한 글 참고하면 좋을 것 같습니다~

gimhanul commented on src/components/Ranking/rankingList.scss:
```diff
@@ -0,0 +1,53 @@
+@import "/src/styles/common.scss";
+
+.ranking-list {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 900px;
+    margin: 20px auto;
+}
+
+.front-info {
+    display: flex;
+    align-items: center;
+}
```
html에서 보면 `.ranking-list > .front-info` 하고 쭉쭉 밑으로 있는 구조인데, 이런 구조에서는 scss의 장점인 중첩을 사용할 수 있습니다.
중첩을 사용하면, 여러 파일이 겹치는데 우연히 클래스 이름이 겹쳐도 버그가 안 납니다.
구글에서 찾아보세용~

SEOKKAMONI committed:
FIX :: 리뷰 반영
- 변수 카멜케이스로 수정
- scss 중첩 적용
